.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+version: 2.1
+setup: true
+orbs:
+  orb-tools: circleci/orb-tools@11.5
+  shellcheck: circleci/shellcheck@3.1
+
+filters: &filters
+  tags:
+    only: /.*/
+
+workflows:
+  lint-pack:
+    jobs:
+      - orb-tools/lint:
+          filters: *filters
+      - orb-tools/pack:
+          filters: *filters
+      - orb-tools/review:
+          filters: *filters
+      - shellcheck/check:
+          filters: *filters
+      - orb-tools/publish:
+          orb-name: cypress-io/cypress
+          enable-pr-comment: false
```

Comment:
// NOTE: I had to disable this (which is defaulted to true) because the `orb-tools/publish` job wasn't able to authenticate with GitHub using the `GITHUB_TOKEN` assigned in the CircleCi context